docs/helm-chart/manage-dags-files.rst

```diff
@@ -71,12 +71,22 @@ Finally, update the Airflow pods with that image:
 
 If you are deploying an image with a constant tag, you need to make sure that the image is pulled every time.
 
+Warning :warning:
+
+*Using constant tag should be used only for testing/development purpose. It is a bad practice to use the same tag as you'll lose the history of your code.*
+
 .. code-block:: bash
 
     helm upgrade --install airflow apache-airflow/airflow \
       --set images.airflow.repository=my-company/airflow \
       --set images.airflow.tag=8a0da78 \
       --set images.airflow.pullPolicy=Always
+      
+You need then to refresh pods that use this image. Deleting them as follow will trigger their replacement.
+
+.. code-block:: bash
+
+  "kubectl" delete pod "<airflow-webserver>" "<airflow-scheduler>" "<airflow-worker>"
```

Comment:
```suggestion
      --set images.airflow.pullPolicy=Always
      --set airflowPodAnnotations.random=r${RANDOM}
```

You'd be much better off doing this instead, or using `openssl rand -hex 8`, or something (you get the idea).

This will recreate all of the pods without requiring manual intervention, but for sure heed @potiuk's warning.